docs/built-in-models/linear.md

```diff
@@ -80,3 +82,21 @@ We can then plot the decision boundary of the classifier and compare those resul
 ![Sklearn model decision boundaries](../figures/logistic_regression_clear.png) ![FHE model decision boundarires](../figures/logistic_regression_fhe.png)
 
 The overall accuracy scores are identical (93%) between the scikit-learn model (executed in the clear) and the Concrete ML one (executed in FHE). In fact, quantization has little impact on the decision boundaries, as linear models are able to consider large precision numbers when quantizing inputs and weights in Concrete ML. Additionally, as the linear models do not use PBS, the FHE computations are always exact. This means that the FHE predictions are always identical to the quantized clear ones.
+
+## Loading a pre-trained model
+
+An alternative to the example above is to train a scikit-learn model in a separate step and then to convert it to Concrete ML.
+
+<!--pytest-codeblocks:cont-->
+
+```
+from sklearn import LogisticRegression as SKlearnLogisticRegression
+
+# Instantiate the model:
+model = SKlearnLogisticRegression()
+
+# Fit the model:
+model.fit(X_train, y_train)
+
+cml_model = LogisticRegression(model, X_train, n_bits=8)
+```
```

Comment:
also maybe I would add the compile + predict steps to better show that this does not replace other steps (like the compile one) , as you wish